File: src/main/java/com/data/utils/GuildUtils.java
package com.data.utils;

import com.mewna.catnip.entity.guild.Guild;
import com.mewna.catnip.entity.guild.Role;

public class GuildUtils {

    public static Role getHighestRole(Guild guild) {
        Role highest = null;
        for ( Role r : guild.roles()) {
            if (highest == null)
                highest = r;
            else if (highest.position() < r.position())
                highest = r;
        }
        return highest;
    }
}

File: src/main/java/com/data/utils/GuildUtils.kt
package com.data.utils

import com.mewna.catnip.entity.guild.Guild
import com.mewna.catnip.entity.guild.Role

object GuildUtils {
    @JvmStatic
    fun getHighestRole(guild: Guild): Role? {
        var highest: Role? = null
        for (r in guild.roles()) {
            if (highest == null) highest = r else if (highest.position() < r.position()) highest = r
        }
        return highest
    }
}